### PR TITLE
fix: Update trustbloc resolve url

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: test/bdd/pkg/context/context.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func NewBDDContext(caCertPath, testDataPath string) (*BDDContext, error) {
 		return nil, err
 	}
 
-	vdri, err := createVDRI("http://localhost:8080/1.0/identifiers")
+	vdri, err := createVDRI("http://localhost:8080/1.0/identifiers", "http://localhost:8072/1.0/identifiers")
```
Comment (troyronda) [inline test/bdd/pkg/context/context.go:43]: some constants would make that more clear :).

Comment (rolsonquadras) [reply test/bdd/pkg/context/context.go]: [Done](https://github.com/trustbloc/edge-service/compare/20d35c6e19eab6192e155c79da794478da86e291..4585fcb01eb809caf81f16aa0068926c2de077f0).

File: test/bdd/pkg/context/context.go [hunk 2]
```diff
@@ -70,7 +75,7 @@ func NewBDDContext(caCertPath, testDataPath string) (*BDDContext, error) {
 	return &instance, nil
 }
 
-func createVDRI(universalResolver string) (vdriapi.Registry, error) {
+func createVDRI(universalResolver, trustBlocResolverURL string) (vdriapi.Registry, error) {
```
Comment (fqutishat) [inline test/bdd/pkg/context/context.go:78]: can you remove universalResolver arg

Comment (fqutishat) [reply test/bdd/pkg/context/context.go:78]: not used in the method

Comment (rolsonquadras) [reply test/bdd/pkg/context/context.go:78]: It's used by non-trustbloc DID methods https://github.com/trustbloc/edge-service/blob/master/test/bdd/pkg/context/context.go#L74-L78.

Comment (rolsonquadras) [reply test/bdd/pkg/context/context.go]: [Updated](https://github.com/trustbloc/edge-service/compare/a05a8f4c9757ffceeb61e617ab20a4741467469f..0e4f5f2d43393ddb3762060c2763c68efd68e15e)

**Pull request merged**